openapi/components/schemas/GatewayAccountConfig/DataCash.yaml

```
@@ -62,4 +62,13 @@ allOf:
             minimum: 0
             default: 0
       threeDSecureServer:
-        $ref: ../ThreeDSecureServers/DataCash3dsServers/DataCash3dsServers.yaml
+        type: object
+        description: DataCash 3DS servers.
+        required:
+          - name
+        properties:
+          name:
+            description: Merchant plug-in name.
```

suggestion: suggestion for readability (series of nouns)
```suggestion
            description: Name of the merchant plug-in.
```